conda/history.py

```diff
@@ -89,13 +89,17 @@ def update(self):
                 return
             curr = set(linked(self.prefix))
             if last == curr:
+                # print a head when a blank env is first created to preserve history
+                if enter_or_exit == 'exit' and not self.parse():
```

Comment:
`self.parse()` is a pretty heavy function for just needing to know if there's anything in the history file.  I wonder if there's an easier way to get this information?